### PR TITLE
Shalliburton/caseflow 394 add post remand appeal info

### PR DESCRIPTION
**Pull request by sahalliburton**
Resolves [CASEFLOW-394](https://vajira.max.gov/browse/CASEFLOW-394)

### Description
Adds the CAVC remand link to source appeal

### Acceptance Criteria
- [ ] Code compiles correctly

### Testing Plan

**Setup**
1. Open a rails console and find the last CAVC Remand: `appeal=CavcRemand.last.remand_appeal`
2. Find the appeals distribution task and record its id: `appeal.treee`
3. Set the parent task to be the distribution task found by its ID listed from the above command: `parent=DistributionTask.find(task_id)`
4. Finally create a schedule hearing task so that you can see the appeal information when scheduling this veteran: `ScheduleHearingTask.create!(appeal: appeal, parent: parent)`

**General**
1. Shadow user `BVASYELLOW`: http://localhost:3000/test/users
2. Navigate to the case details page for the appeal you modified above
3. On the case details page, click the 'Schedule Veteran' action in the dropdown
4. Ensure that the appeal information is displayed correctly on the left
5. Ensure that you can still schedule the veteran successfully

### CODE REVIEW

File: app/models/serializers/work_queue/appeal_serializer.rb [hunk 1]
```diff
@@ -80,6 +80,16 @@ class WorkQueue::AppealSerializer
 
   attribute :cavc_remand
 
+  attribute :remand_source_appeal_id do |appeal|
+    if !appeal.cavc_remand.nil?
```
Comment (rubaiyat22) [inline app/models/serializers/work_queue/appeal_serializer.rb:84]: nitpick - take it or leave it!
```suggestion
    if appeal.cavc_remand.present?
```

Comment (sahalliburton) [reply app/models/serializers/work_queue/appeal_serializer.rb:84]: Thanks! I always forget about those ruby shortcuts

File: app/models/serializers/work_queue/appeal_serializer.rb [hunk 2]
```diff
@@ -80,6 +80,16 @@ class WorkQueue::AppealSerializer
 
   attribute :cavc_remand
 
+  attribute :remand_source_appeal_id do |appeal|
+    if appeal.cavc_remand.present?
+      Appeal.find(appeal.cavc_remand.source_appeal_id).uuid
+    end
```
Comment (tomas-nava) [inline app/models/serializers/work_queue/appeal_serializer.rb:86]:
```suggestion
    appeal.cavc_remand&.source_appeal&.uuid
```

**Review by tomas-nava — APPROVED** [review]
approve! with one non-blocking suggestion

**Pull request merged**